packages/connect-query/src/create-use-query-options.ts

```diff
@@ -127,6 +127,6 @@ export function createUseQueryOptions<
       callOptions,
     }),
     structuralSharing,
-    enabled: input === disableQuery ? false : undefined,
+    ...(input === disableQuery ? { enabled: false } : undefined),
```

Comment:
Changing the logic: Instead of returning `enabled: undefined` from our option factories, we omit the property. 

This means we can use the spread operator to merge our options with user-provided options (see below).

Reply:
The reason this was kept undefined was that enabled can be provided as a default option at the top level (when providing default settings for all queries). This was left as undefined to respect that default. 

Otherwise, yeah spread would've made sense.

Reply:
By returning `undefined`, we do exactly the opposite: We return a property that would override the query client defaults. Then [we undo it](https://github.com/connectrpc/connect-query-es/blob/v1.4.2/packages/connect-query/src/use-query.ts#L69-L71) in `useClient`.

Maybe I'm missing something. Can you elaborate?

Reply:
Actually, nvm, this is what I get for reviewing code on my phone. I understand where I got mixed up.